career-link-webapi/Controllers/PostController.cs

```diff
@@ -13,16 +15,80 @@ public PostController(IPostService taskService)
             _postService = taskService;
         }
 
-        [HttpGet("allpost")]
+        [HttpGet]
         public async Task<IActionResult> GetAllTask()
         {
             return Ok(await _postService.GetPosts());
         }
-        
+
         [HttpGet("{id}")]
         public async Task<IActionResult> GetTaskByID(int id)
         {
             return Ok(await _postService.GetPostById(id));
         }
+
+        [HttpGet("search")]
+        public async Task<ActionResult<List<PostDTO>>> SearchPost([FromQuery] int experienceLevel, [FromQuery] List<int> skillIds)
```

Comment:
this endpoint is getting list of skillIds??

Reply:
list of skill names